### PR TITLE
Remove deprecated and NEXT_MAJOR code

### LINKED ISSUE #5763: [Next Major] Remove deprecated code

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is a BC break.

Closes #5763

## Changelog

```markdown
### Removed
- Remove deprecated and NEXT_MAJOR code
```

### CODE REVIEW

File: src/Admin/AbstractAdmin.php [hunk 1]
```diff
@@ -454,6 +405,11 @@ abstract class AbstractAdmin implements AdminInterface, DomainObjectInterface, A
      */
     protected $accessMapping = [];
 
+    /**
+     * @var string|array
```
Comment (OskarStark) [inline src/Admin/AbstractAdmin.php]: String? 🧐

File: src/Admin/AbstractAdmin.php [hunk 2]
```diff
@@ -1008,37 +920,19 @@ public function getActiveSubClass()
...
-                __CLASS__
-            ), E_USER_DEPRECATED);
-            // NEXT_MAJOR : remove the previous `trigger_error()` call, the `return null` statement, uncomment the following exception and declare string as return type
-            // throw new \LogicException(sprintf(
-            //    'Admin "%s" has no active subclass.',
-            //    static::class
-            // ));
-
-            return null;
+            throw new \LogicException(sprintf(
```
Comment (OskarStark) [inline src/Admin/AbstractAdmin.php]: We should go here with our own LogicException and I really would love to see the way doctrine uses exceptions: named ctors, like:

LogicException::noActiveSubClass($class)

WDYT ?

File: src/Admin/AbstractAdmin.php [hunk 3]
```diff
@@ -2523,9 +2270,7 @@ public function getDashboardActions()
             $actions['create'] = [
                 'label' => 'link_add',
                 'translation_domain' => 'SonataAdminBundle',
-                // NEXT_MAJOR: Remove this line and use commented line below it instead
-                'template' => $this->getTemplate('action_create'),
-                // 'template' => $this->getTemplateRegistry()->getTemplate('action_create'),
+                 'template' => $this->getTemplateRegistry()->getTemplate('action_create'),
```
Comment (OskarStark) [inline src/Admin/AbstractAdmin.php]: Indention looks wrong

File: src/Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php [hunk 4]
```diff
@@ -62,9 +62,10 @@ final class AdminExtractor implements ExtractorInterface, TranslatorInterface, S
      */
     private $breadcrumbsBuilder;
 
-    public function __construct(Pool $adminPool, LoggerInterface $logger = null)
+    public function __construct(Pool $adminPool, BreadcrumbsBuilderInterface $breadcrumbsBuilder, LoggerInterface $logger = null)
```
Comment (OskarStark) [inline src/Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php]: Hmm the order change of the arguments should be deprecated first IMO

**Comment by core23** [comment]
I'm splitting this PR, because removing too much is to complicated...

**Pull request closed without merging**